comp/core/workloadmeta/collectors/internal/kubelet/kubelet.go

```diff
@@ -309,6 +309,7 @@ func (c *collector) parsePodContainers(
 				State:           containerState,
 				Owner:           parent,
 				Resources:       resources,
+				RestartCount:    container.RestartCount,
```

Comment:
Should be removed as the `container.RestartCount` does not apply to this container id but to the container name